### PR TITLE
Update eventing

### PR DESCRIPTION
**Pull request by slinkydeveloper**
Signed-off-by: Francesco Guardiani <francescoguard@gmail.com>

Fixes #1300

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -161,6 +161,8 @@ application. This Go file should use the package `main` and include the function
 `main()`. It is a good practice to include a `readme` file as well. When
 uploading test images, `ko` will build an image from this folder.
 
+If you need to add a new test image imported from eventing, import it in `hack/tools.go`
```
Comment (mattmoor) [inline test/README.md:164]: Format markdown:
```suggestion
If you need to add a new test image imported from eventing, import it in
`hack/tools.go`
```

File: test/README.md [hunk 2]
```diff
@@ -161,6 +161,9 @@ application. This Go file should use the package `main` and include the function
 `main()`. It is a good practice to include a `readme` file as well. When
 uploading test images, `ko` will build an image from this folder.
 
+If you need to add a new test image imported from eventing, import it in
```
Comment (mattmoor) [inline test/README.md:164]: Remove trailing whitespace:
```suggestion
If you need to add a new test image imported from eventing, import it in
```

Comment (mattmoor) [inline test/README.md:164]: Format markdown:
```suggestion
If you need to add a new test image imported from eventing, import it in
```

File: test/README.md [hunk 3]
```diff
@@ -161,6 +161,9 @@ application. This Go file should use the package `main` and include the function
...
+`hack/tools.go`
+
```
Comment (aliok) [inline test/README.md:166]: :+1:

**Pull request merged**